070-Integrations/024-cloudflare-workers.mdx

```diff
@@ -0,0 +1,268 @@
+---
+title: Get started with Cloudflare and Xata
+navTitle: Cloudflare Workers
+keywords: ['cloudflare', 'full-text search']
+description: Use Cloudflare and Xata
+slug: getting-started/cloudflare
+published: true
+---
+
+You will need a cloudflare account to deploy your application.
+
+The completed [Cloudflare and Xata code](https://github.com/xataio/examples/tree/main/apps/getting-started-cloudflare-workers) for this
+guide is available via the [Xata `examples` repo](https://github.com/xataio/examples) on GitHub.
+
+## Before you begin
+
+Install the Xata CLI:
+
+```sh
+npm install -g @xata.io/cli
+```
+
+Authenticate the Xata CLI with your Xata account. If you don't already have an account, you can use the
+same workflow to sign up for a new account. Run the following command to begin the authentication workflow:
+
+```sh
+xata auth login
+```
+
+On completion, the command will create a new API key for your user account, which you should see in the \[account
+settings page within the Xata UI]\[1]. That key will also be stored locally on your computer (the location might vary for
+each OS). It looks like this:
+
+```toml
+# .config/xata/credentials
+[default]
+apiKey=YOUR_API_KEY_HERE
+```
+
+## Create a new Cloudflare app
+
+1.  Create a new Cloudflare workers endpoint, accepting the default prompt options:
+
+    ```sh
+    npm create cloudflare@latest
+    ```
+
+2.  Once the command has completed, go to the `getting-started-cloudflare-workers` directory, and run the application:
+
+    ```sh
+    $ npm run wrangler dev
+    ```
+
+By default, the application will run on `http://localhost:8787`. When visiting this URL "Hello World" should be shown in your browser.
+
+## Create a new database
```

Comment:
This section should probably go, would be great to offload it to a link. I think the CSV import stuff should definitely go